code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -323,49 +340,50 @@
 
 
 /mob/living/carbon/human/ex_act(severity, ex_target)
-	var/b_loss = null
-	var/f_loss = null
-	var/bomb_armor = getarmor(null, "bomb")
+
 	if(istype(ex_target, /datum/spacevine_mutation) && isvineimmune(src))
 		return
+	var/b_loss = 0
+	var/f_loss = 0
+	var/bomb_armor = getarmor(null, "bomb")
 
 	switch (severity)
 		if (1)
-			b_loss += 500
-			if (prob(bomb_armor))
-				shred_clothing(1,150)

```

Comment:
we don't need to shred clothes here, ex_act() calls ..() all the way down to atom/ex_act() which calls contents_explosion() which calls ex_act() on all the clothes.